typescript/styling-engine.ts

```diff
@@ -1,14 +1,19 @@
 import { HeadingElement } from './forms'
 
-export type FormStyle = BackgroundStyle & {
-  /** The font family to use for the entire form page */
+export type FormsAppStyle = {
```

Comment:
Should there be a new property on the environment that is this type?

Reply:
Gahh yep.